pip/poetry/poetry_unit_test.go

```diff
@@ -0,0 +1,68 @@
+package poetry
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const testDataPath = "testdata"
+
+func setupPoetry(path string) *Poetry {
+	poetry := New()
+	err := poetry.SetRootModule(path)
+	if err != nil {
+		panic(err)
+	}
+
+	return poetry
+}
+
+func TestSetRootModule(t *testing.T) {
+	poetry := New()
+	err := poetry.SetRootModule(testDataPath)
+	assert.Nil(t, err)
+	assert.Equal(t, "testdata", poetry.basepath)
+}
+
+func TestGetVersion(t *testing.T) {
+	poetry := setupPoetry(testDataPath)
```

Comment:
After modifying setupPoetry ass shown above pass the context here:
```suggestion
	poetry := setupPoetry(t, testDataPath)
```